com.unity.ml-agents/Runtime/Sensor/RayPerceptionSensorComponentBase.cs

```diff
@@ -130,14 +130,14 @@ public void OnDrawGizmos()
                 // hit fraction ^2 will shift "far" hits closer to the hit color
                 var lerpT = rayInfo.hitFraction * rayInfo.hitFraction;
                 var color = Color.Lerp(rayHitColor, rayMissColor, lerpT);
-                color.a = alpha;
+                color.a *= alpha;
```

Comment:
Ran into this when trying to get a screenshot; I wanted to hide the "higher" raycast component by setting its alpha to 0